enterprise/releases/2023-07-0.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: '2023.07.0 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Server: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2023.07.0`
+
+## Notes
+
+- This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+
+## Features
+
+* Index Advisor
+  - Support analyzing queries that require the "ltree" extension <!-- PR 2913 -->
```

Comment:
This looks bad/annoying in GitHub. I wonder if `{/* PR 2913 */}`is less annoying here 🤔 (not sure how GitHub will treat it)

Reply:
Unfortunately `{/* PR 2913 */}` isn't a valid comment syntax in MDX - its weird why GitHub doesn't render this correctly, given that this is how you're supposed to do comments in Markdown.

For some additional context, I added the PR numbers here to allow us to better understand how we could utilize our new changelog data to semi-automate, or potentially cross reference the Enterprise release notes to the overall changelog. I'll leave these in for now in the `<!-- PR 2913 -->` comment syntax to help with exploring whether this is actually useful.